dev/statement_serializer.h

```diff
@@ -1031,13 +1051,11 @@ namespace sqlite_orm {
                 if(!context.skip_types_and_constraints) {
                     ss << " " << type_printer<field_type_t<column_type>>().print();
                     const bool columnIsNotNull = column.is_not_null();
-                    auto constraintsTuple = streaming_column_constraints(
-                        call_as_template_base<column_constraints>(polyfill::identity{})(column),
-                        columnIsNotNull,
-                        context);
-                    if(std::tuple_size<decltype(constraintsTuple)>::value > 0) {
-                        ss << " " << constraintsTuple;
-                    }
+                    ss << " "
+                       << streaming_column_constraints(
```

Comment:
I put `streaming_column_constraints` back into `operator<<` statement not variable declaration to make it right. But `const bool columnIsNotNull = column.is_not_null();` is left as a constant in stack cause it has to have real memory address

Reply:
It already has a "real memory address", in the form of an rvalue, so there's no need for another variable.
Like I mentioned in my [comment](https://github.com/fnc12/sqlite_orm/pull/1223/files#r1353270989), the only problem is if you would capture that rvalue in a variable that outlives the temporary rvalue.
That very rvalue will live for the whole streaming expression.

Reply:
It already has a "real memory address", in the form of an rvalue, so there's no need for another variable.
Like I mentioned in my [comment](https://github.com/fnc12/sqlite_orm/pull/1223/files#r1353270989), the only problem is if you would capture that rvalue in a variable that outlives the temporary rvalue.
That very rvalue will live for the whole streaming expression.